docs/modules/admin/pages/distributed_install.adoc

```diff
@@ -39,33 +43,39 @@ Add `opennms-kafka-producer` to the `featuresBoot` property in the `$OPENNMS_HOM
 
 === Enable consumption of events from Kafka
 
-Enable the Sink API consumer for Eventd:
+.Enable the Sink API consumer for eventd:
+
 ```
 echo 'org.opennms.netmgt.eventd.sink.enable=true' > "$OPENNMS_HOME/etc/opennms.properties.d/event-sink.properties"
 ```
 
-IMPORTANT: We assume that you're already using the Kafka strategy for the Sink API. See <<prequisites>>.
+IMPORTANT: We assume that you are already using the Kafka strategy for the Sink API.
+See <<prequisites>>.
 
 == Install ALEC in OpenNMS
 
-On the OpenNMS server, install the package:
+.On the OpenNMS server, install the package:
+
 ```
 yum -y install opennms-alec-plugin
 ```
 
-Configure the feature set to be installed when OpenNMS starts:
+.Configure the feature set to be installed when OpenNMS starts:
+
 ```
 echo 'alec-opennms-distributed wait-for-kar=opennms-alec-plugin' > "$OPENNMS_HOME/etc/featuresBoot.d/alec.boot"
 ```
 
 == Install ALEC in Sentinel
 
-On the Sentinel systems, install the package:
+.On the Sentinel systems, install the package:
+
 ```
 yum -y install sentinel-alec-plugin
 ```
 
-Configure the Kafka Stream (consumer):
+.Configure the Kafka Stream (consumer):
+
 ```
 echo 'bootstrap.servers=127.0.0.1:9092
 commit.interval.ms=5000' > "$SENTINEL_HOME/etc/org.opennms.alec.datasource.opennms.kafka.streams.cfg"
```

Comment:
This can be made into a source block using the syntax from my previous suggestions. I'd make the change, but I can't suggest edits for the final line of the snippet.